test/rack/prx_auth/token_data_test.rb

```diff
@@ -48,5 +48,59 @@
       assert !token.authorized?(789)
     end
 
+    describe 'with wildcard role' do
+      let(:aur) { {'*' => 'peek', '123' => 'admin', '456' => 'member' } }
+
+      it 'applies wildcard tokens to queries with no matching aur' do
+        assert token.authorized?(789, :peek)
```

Comment:
What happens if you ask for `:peek` on 123 or 456?

Reply:
https://github.com/PRX/rack-prx_auth/pull/18/files/9d8df21f38566f7a0e9d193e102432bc512939b1#diff-7c569a6167ee73eb7b1bb010dfcaf2a8R72

Reply:
Makes sense!